config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -164,3 +164,26 @@ periodics:
     testgrid-alert-email: release-managers+alerts@kubernetes.io
     testgrid-dashboards: sig-release-releng-informing
     testgrid-tab-name: build-packages-rpms
+
+- interval: 6h
+  name: ci-fast-forward
+  cluster: k8s-infra-prow-build
```

Comment:
Can we submit GCB jobs to the kubernetes-release-test project from this cluster?

Reply:
i think we need to have a service account for that, similar we have `gcb-builder-releng-test`

but to be 100% I think @ameukam can help us here

Reply:
@cpanato @saschagrunert Sorry for missing this. Service account `gcb-builder@k8s-infra-prow-build-trusted.iam.gserviceaccount.com` is allowed to trigger GCB jobs for all the staging projects. The job need to run on `k8s-infra-prow-build-trusted`.

Reply:
thanks so much Arnaud for the clarification! i always get confused with those things and you are always here to help me :)

Reply:
Fixed, thank you both @ameukam and @cpanato.